src/golang/cmd/server/handler/v2/dag_results_get.go

```diff
@@ -60,9 +69,21 @@ func (h *DAGResultsGetHandler) Prepare(r *http.Request) (interface{}, int, error
 		return nil, http.StatusBadRequest, err
 	}
 
+	limit, err := (parser.LimitQueryParser{}).Parse(r)
+	if err != nil {
+		return nil, http.StatusBadRequest, err
+	}
+
+	orderBy, err := (parser.OrderByQueryParser{}).Parse(r, models.AllDAGResultCols())
+	if err != nil {
+		return nil, http.StatusBadRequest, err
+	}
+
 	return &dagResultsGetArgs{
 		AqContext:  aqContext,
 		workflowID: workflowID,
+		orderBy:    orderBy,
```

Comment:
I slightly prefer to make `DESC` vs `ASC` more explicit. We can refine towards more generalized way later (e.g. use a structure to capture all DB related query parameters), but for now, it would be helpful to simply rename `order_by` to `order_by_desc`

Reply:
Added a desc/asc parameter